doc/operations/certificates.rst

```diff
@@ -98,8 +98,14 @@ To be able to bind to that port, the ``cchost`` binary may need to be given spec
 
```

Comment:
Can't comment on the _actual_ line (1 above), but should we change this from `json` to `python`, so we can include comments, snips, multi-line strings?

Reply:
Done